src/azure-cli-core/azure/cli/core/profiles/__init__.py

```diff
@@ -3,11 +3,23 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-from azure.cli.core.profiles._shared import AZURE_API_PROFILES
+from azure.cli.core.profiles._shared import (AZURE_API_PROFILES,
+                                             get_api_version as _sdk_get_api_version,
+                                             get_versioned_models as _sdk_get_versioned_models)
 
 # API Profiles currently supported in the CLI.
 API_PROFILES = {
-    'latest': {},
-    '2016-01-01-example': AZURE_API_PROFILES['2016-01-01-example'],
-    '2015-06-15-example': AZURE_API_PROFILES['2015-06-15-example']
+    'latest': AZURE_API_PROFILES['latest'],
+    '2016-example': AZURE_API_PROFILES['2016-example'],
+    '2015-example': AZURE_API_PROFILES['2015-example']
 }
+
+
+def get_api_version(resource_type):
```

Comment:
I'm fine with that, but we need to abstract out the API version as much as possible so we don't have "get_api_version(...)" all over the place.